command/genesis/polybft_params.go

```diff
@@ -361,7 +361,7 @@ func (p *genesisParams) generatePolyBftChainConfig(o command.OutputFormatter) er
 	if p.isBurnContractEnabled() {
 		// only populate base fee and base fee multiplier values if burn contract(s)
 		// is provided
-		chainConfig.Genesis.BaseFee = command.DefaultGenesisBaseFee
+		chainConfig.Genesis.BaseFee = p.baseFee
 		chainConfig.Genesis.BaseFeeEM = command.DefaultGenesisBaseFeeEM
```

Comment:
Why don't allow flag for `BaseFeeEM` also? Maybe we should allow `--genesis-base-fee` flag to specify both values like `baseFee[:baseFeeEM]` (two values separated with :, by default `baseFeeEM` is not required). what do you think?

Reply:
That's a really good point, will change the code accordingly.